src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java

```diff
@@ -15,22 +10,14 @@
 import org.springframework.ui.Model;
 import org.springframework.web.bind.annotation.*;
 
-import com.google.common.collect.ImmutableMap;
-
-import ca.corefacility.bioinformatics.irida.model.joins.Join;
-import ca.corefacility.bioinformatics.irida.model.joins.impl.ProjectSampleJoin;
 import ca.corefacility.bioinformatics.irida.model.project.Project;
 import ca.corefacility.bioinformatics.irida.model.sample.Sample;
-import ca.corefacility.bioinformatics.irida.model.sample.SampleSequencingObjectJoin;
-import ca.corefacility.bioinformatics.irida.model.sequenceFile.SequenceFile;
 import ca.corefacility.bioinformatics.irida.ria.web.cart.components.Cart;
 import ca.corefacility.bioinformatics.irida.ria.web.cart.dto.*;
 import ca.corefacility.bioinformatics.irida.service.ProjectService;
 import ca.corefacility.bioinformatics.irida.service.SequencingObjectService;
 import ca.corefacility.bioinformatics.irida.service.sample.SampleService;
 import ca.corefacility.bioinformatics.irida.service.user.UserService;
```

Comment:
It looks like both `UserService` and `SequencingObjectService` are unused in this class now. Could you remove those?

Reply:
Thanks, updated in 4df5132465c06c43fa35c81b19c454c0880efb7e